### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by sombit-g**
Changed alignment and improved some grammar in the sentences

__Which issue does this PR address?__

Closes #

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [x] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/.github/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Please explain how those changes are improvements to the existing code base.

**Comment by sombit-g** [comment]
Hi, so I was just going through the documents when I came across some grammar errors. These errors, if fixed wil make the document even more readable and make it sound better. Thanks

File: README.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ Welcome to docs.bpmn.io. This repository contains documentation for:
 
  - diagram-js
  - bpmn-js
- - dmn-js
-
-libraries.
+ - dmn-js libraries.
```
Comment (pinussilvestrus) [inline README.md]: I wouldn't remove the new line here. It's an enumeration, therefore the ending phrase should be below.

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]
One minor thought commented. Also, please update the commit message to fit our guideline

> chore(README): fix typos and grammatical errors

File: README.md (hunk 1 above)
Comment (sombit-g) [reply README.md]: Ok. I thought it might look better that way. No problem I will make the changes. Any other changes you suggest?

**Comment by sombit-g** [comment]
I have made the necessary changes as suggested. Any other suggestions?

**Comment by pinussilvestrus** [comment]
Still not a valid pull request according to our Definition of Done.

**Comment by philippfromme** [comment]
We do not accept PRs that do not substantially improve our toolkit. For potential improvements check out the open issues of this repository.

**Pull request closed without merging**